src/plugins/expressions/common/expression_types/specs/vis_layers.ts

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
There should be no need to add this expression type to the `expressions` plugin itself. Instead, expression types can be registered just like expression functions, from the `vis_augmenter` plugin. For example, see https://github.com/opensearch-project/OpenSearch-Dashboards/blob/100a41ac9a7afb0807ad76913696f2a9d0f75ecb/src/plugins/data/public/search/search_service.ts#L99-L100 

It will also be nicer to have all this stuff co-located.

Reply:
Great point. I had originally added there when the project scope wasn't fully defined, I'll relocate to the `vis_augmenter` plugin 👍